spec/api-browser-window-spec.js

```diff
@@ -1106,6 +1106,90 @@ describe('BrowserWindow module', function () {
       })
       w.loadURL('file://' + path.join(fixtures, 'api', 'close-beforeunload-empty-string.html'))
     })
+
+    it('fires for each close attempt', function (done) {
+      var closeCount = 5
+
+      var timeoutHandle = setTimeout(() => {
+        w.webContents.executeJavaScript('isDone = true')
```

Comment:
`executeJavaScript` is an async operation, spec should be ended once its resolved.